components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -801,6 +802,10 @@ class PageWallet extends React.Component<Props, State> {
     const walletType = externalWallet ? externalWallet.type : undefined
     const walletProvider = utils.getWalletProviderName(externalWallet)
 
+    if (!paymentId || paymentId.length === 0) {
```

Comment:
This can just be `!paymentId` (JS will cast an empty string to false).

Reply:
Thanks, fixed!